benchmark/benchmarkTests.go

```diff
@@ -89,7 +89,8 @@ func SociFullRun(
 	imageRef string,
 	indexDigest string,
 	readyLine string,
-	testName string) {
+	testName string,
+	parseFileAccessPatterns bool) {
```

Comment:
Since the collection of `containerStartTime` happens within the parser itself (see : https://github.com/awslabs/soci-snapshotter/pull/218#discussion_r1041388504) this optional parameter is no longer needed. The running of the parser happens as a part of the `benchmark.Run()` call as the parser is now assigned to the `AfterFunction` attribute of the test driver which will execute a given function after each image is fully ran.

Reply:
Something like:


```
opts = []bparser.ParseOpts{bparser.WithParseFileAccessPaterns,bparser.WithParseLifecycleOperations}
...
driver := framework.BenchmarkTestDriver{
...
AfterOpts: opts
...
}
```